Cone_spanners_2/test/Cone_spanners_2/CMakeLists.txt

```diff
@@ -5,12 +5,16 @@ cmake_minimum_required(VERSION 3.1...3.23)
 project(Cone_spanners_2_Tests)
 
 find_package(CGAL REQUIRED COMPONENTS Core)
+find_package(LEDA QUIET)
 
-# create a target per cppfile
-file(
-  GLOB cppfiles
-  RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
-  ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
-foreach(cppfile ${cppfiles})
-  create_single_source_cgal_program("${cppfile}")
-endforeach()
+create_single_source_cgal_program("cones_inexact.cpp")
+create_single_source_cgal_program("theta_inexact.cpp")
+create_single_source_cgal_program("yao_inexact.cpp")
+
+if(CGAL_Core_FOUND OR LEDA_FOUND)
```

Comment:
That is strange. `find_package(CGAL REQUIRED COMPONENTS Core)` should stop the compilation if `Core` is not found. That is certainly an error in `CGALConfig.cmake`.

Reply:
We can open an independent issue if we want to fix that. OPTIONAL keyword should indeed be used.